packages/scss/src/components/comment/component.scss

```diff
@@ -73,26 +95,18 @@
 			font-size: var(--components-comment-text-fontSize);
 			line-height: var(--components-comment-text-lineHeight);
 		}
+	}
+}
 
-		.comment-content-textContainer {
-			font-size: var(--components-comment-text-fontSize);
-			line-height: var(--components-comment-text-lineHeight);
-		}
-
-		.commentWrapper {
-			display: flex;
-			flex-direction: column;
-			gap: var(--pr-t-spacings-100);
-			margin: 0;
-			padding: 0;
-		}
+@mixin chat {
+	@include reset.list;
 
-		.commentWrapper-item {
-			list-style: none;
+	display: flex;
+	flex-direction: column;
+	gap: var(--pr-t-spacings-200);
 
-			&.mod-WrapperAvatar ~ &:not(.mod-WrapperAvatar) {
-				padding-left: var(--components-comment-content-margin);
```

Comment:
Is there a reason behind this deletion? 
It seems to break the comment's wrap (with avatar only on first comment) :
![image](https://github.com/user-attachments/assets/bea05204-fde0-4ac2-ae6b-60db351fcdcf)